src/invertObj.js

```diff
@@ -5,6 +5,9 @@ var keys = require('./keys');
 /**
  * Returns a new object with the keys of the given object
  * as values, and the values of the given object as keys.
+ * Note that the last key found is prefered when handling
+ * the same value. Non-primitive values are converted to
+ * strings which might yield undesired results.
```

Comment:
Thanks for the corrections!
- Removed the *undesired results*
- Added the *coercion to string* in the first sentence to avoid repetition